routes/routes_test.go

```diff
@@ -11,7 +11,17 @@ import (
 )
 
 func TestRoutes(t *testing.T) {
-	routes := []string{"AddTask", "Status", "ListRunningTasks"}
+	routes := []string{
```

Comment:
maybe crazy idea. what if base this list on `routes()` and we never miss a route :fireworks: 

If we add an assert about the length we can make sure it doesn't silently break